cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -113,10 +113,11 @@ jobs:
       - id: plan
         run: |
           cargo dist
-          {{%- if dispatch_releases %}} ${{ inputs.tag && (inputs.tag != 'dry-run' && format('host --steps=create --tag={0}', inputs.tag) || format('host --steps=create', inputs.tag))
-          {{%- else %}} ${{ !github.event.pull_request && format('host --steps=create --tag={0}', github.ref_name)
+          {{%- if dispatch_releases %}} ${{ (inputs.tag && inputs.tag != 'dry-run' && format('host --steps=create --tag={0}', inputs.tag))
```

Comment:
I feel like the conditionals here are getting complex enough it might be nice to comment on it; if we run into bugs here it'll help us remember our reasoning a little better.

Reply:
that's a good call, i'll add some comments in a follow up (rn just wanna get the fix out)